test/session-pool.ts

```diff
@@ -1367,34 +1352,12 @@ describe('SessionPool', () => {
   });
 
   describe('trace annotations on active span', () => {
```

Comment:
maybe for separate PR. But It will be good to extract these tests in `observability-test` folder